### PR TITLE
keep default effects for file in groups

### PR DESCRIPTION
**Pull request by dmitry-mukhin**
addressing #66

### CODE REVIEW

**Comment by homm** [comment]
PHP 7.0 is released. Please, add to `travis.yml`.

File: tests/ApiTest.php [hunk 1]
```diff
@@ -350,7 +350,7 @@ public function testUploadFromString()
       $this->fail('We get an unexpected exception trying to store uploaded file from contents: '.$e->getMessage());
     }
 
-    usleep(500);
+    usleep(500000);
```
Comment (homm) [inline tests/ApiTest.php]: is_ready?

Comment (dmitry-mukhin) [inline tests/ApiTest.php]: nope

Comment (dmitry-mukhin) [inline tests/ApiTest.php]: it's PHP way

**Pull request merged**